client/app/components/viz/VariantInspectCard.vue

```diff
@@ -421,15 +421,15 @@
             </div>
 
 
-          <div class="variant-row" v-if="selectedVariant && selectedVariant.genotypes">
-            <variant-allele-counts-menu
-              v-if="selectedVariantRelationship != 'known-variants' && cohortModel.getModel(selectedVariantRelationship ? selectedVariantRelationship : 'proband').isBamLoaded()"
-              :selectedVariant="selectedVariant"
-              :affectedInfo="cohortModel.affectedInfo"
-              :cohortModel="cohortModel.mode"
-              :relationship="selectedVariantRelationship">
-            </variant-allele-counts-menu>
-          </div>
+          <!--<div class="variant-row" v-if="selectedVariant && selectedVariant.genotypes">-->
```

Comment:
Hi Will -  For tracking history, it would be best to not check in commented code. (I'm guilty of this too).

Reply:
I added a new commit that removes the commented code from the pull request.  This commit happened after I already made some commits for completed Trello board tasks, so there will be 2 additional commits with this requested change